enhancements/declarative-index-config.md

```diff
@@ -587,7 +591,9 @@ community-operators
     │   └── ...
     └── servicemesh.json
 $ rm -r community-operators/servicemesh
-$ opm index create --from=community-operators --tag=docker.io/someothernamespace/new-community-operators:latest
+$ opm generate community-operators
```

Comment:
This example should probably `cat` the generated Dockerfile to show what was created more directly.